packages/media/src/getImageDimensions.js

```diff
@@ -26,7 +26,12 @@ import preloadImage from './preloadImage';
  * @return {Promise} Image dimensions object.
  */
 const getImageDimensions = async (src) => {
-  const img = await preloadImage(src);
+  let img;
+  try {
+    img = await preloadImage(src);
+  } catch (error) {
+    //ignore
+  }
   return {
```

Comment:
If running into the `catch` above, `img` will be `undefined` still, and trying to access `img.naturalWidth` or `img.naturalHeight` will break.

Second, no need to do `catch (errror) {` if `error` is unused. Can just do ` catch {`

I'd probably write the whole thing like this:

```js
try {
    const img = await preloadImage(src);
    return {
      width: img.naturalWidth,
      height: img.naturalHeight,
    };
  } catch {
    return { width: 0, height: 0 };
  }
```

But I am skeptical that we actually need the `catch()` here.

Everywhere `getImageDimensions` is being used there already seem to be some `catch` in place.

Adding one here and returning `width:0, height: 0` would unexpectedly break things in other ways.

**So in short:** I don't think we should add a `catch` here at all.